lib/checks/tables/td-headers-attr.json

```diff
@@ -4,9 +4,13 @@
   "metadata": {
     "impact": "serious",
     "messages": {
-      "pass": "The headers attribute is exclusively used to refer to other cells in the table",
+      "pass": "The headers attribute is exclusively used to refer to other header cells in the table",
```

Comment:
It'd be good to make a similar update to the description and help metadata of the `td-headers-attr` rule (not just the check)